modules/base/src/processors/volumesequencesource.cpp

```diff
@@ -167,15 +190,11 @@ void VolumeSequenceSource::loadFolder(bool deserialize) {
             basis_.updateForNewEntity(*(*volumes_)[0], deserialize);
             information_.updateForNewVolume(*(*volumes_)[0], deserialize);
         }
+    } else {
+        outport_.detachData();
     }
 }
 
-void VolumeSequenceSource::addFileNameFilters() {
-    file_.clearNameFilters();
-    file_.addNameFilter(FileExtension::all());
-    file_.addNameFilters(rf_->getExtensionsForType<VolumeSequence>());
-}
-
 void VolumeSequenceSource::process() {
     if (file_.isModified() || reload_.isModified() || folder_.isModified() ||
```

Comment:
do we need to add reader_.isModified() here?

Reply:
yes